code/game/gamemodes/umbrage/umbrage_minor_abilities.dm

```diff
@@ -45,23 +45,25 @@
 	var/victims = list() //A list of people we've used the bead on recently; we can't drain them again so soon
 
 /datum/action/innate/umbrage/devour_will/IsAvailable()
-	if(!usr || !usr.get_empty_held_indexes())
+	var/mob/living/user = usr
```

Comment:
I learned about the owner variable halfway into last commit. Back to the drawing board!